api/pom.xml

```diff
@@ -22,14 +22,72 @@
 			<type>jar</type>
 		</dependency>
 		
-		<!--  include groovy, but mark as provided since it is included with the logic module -->
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>webservices.rest-omod</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>webservices.rest-omod-common</artifactId>
+		</dependency>
+		
+		<!-- End OpenMRS modules -->
+
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
+			<type>jar</type>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-test</artifactId>
+			<type>jar</type>
+		</dependency>
+		
+		<!--  Include groovy, but mark as provided since it is included with the logic module -->
+		
 		<dependency>
   			<groupId>org.codehaus.groovy</groupId>
   			<artifactId>groovy</artifactId>
   			<version>1.7.6</version>
   			<type>jar</type>
   			<scope>provided</scope>
-		</dependency>	
+		</dependency>
+		
+		<!-- Begin OpenMRS core -->
+
+		<dependency>
+			<groupId>org.openmrs.api</groupId>

```

Comment:
This and the dependency below are duplicates. There is a warning about it when you run the build:

```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.openmrs.module:patientflags-api:jar:1.3.5-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.openmrs.api:openmrs-api:jar -> duplicate declaration of version (?) @ org.openmrs.module:patientflags-api:[unknown-version], /Users/pascal/workspace/openmrs-module-patientflags/api/pom.xml, line 60, column 15
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.openmrs.web:openmrs-web:jar -> duplicate declaration of version (?) @ org.openmrs.module:patientflags-api:[unknown-version], /Users/pascal/workspace/openmrs-module-patientflags/api/pom.xml, line 66, column 15
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```